src/Mono.Android/Mono.Android.projitems

```diff
@@ -133,5 +133,11 @@
       <Id>26</Id>
       <Stable>True</Stable>
     </AndroidApiInfo>
+    <AndroidApiInfo Include="v8.1">
+      <Name>Oreo</Name>
+      <Level>27</Level>
+      <Id>27</Id>
+      <Stable>True</Stable>
```

Comment:
This should be `False`, not `True`, as our binding is not stable.

Reply:
I have no idea why current MSBuild based solution requires duplicate edits. This should be automatically generated from build-tools/scripts/BuildEverything.mk.